Vagrantfile

```diff
@@ -1,36 +1,38 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
+# Vagrantfile will spin up Centos with Archiva
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "liatrio/centos7chefjava"
+  config.vm.box = 'bento/centos-7.2'
 
-   config.vm.network "forwarded_port", guest: 8080, host: 18080
+  config.vm.network 'forwarded_port', guest: 8080, host: 18_080
```

Comment:
Need to change 18_080 back to 18080, curious if the linting tools automatically did this because I've seen it in some of my cookbooks as well.